core/player/src/view/plugins/__tests__/applicability.test.ts

```diff
@@ -245,21 +248,26 @@ describe("applicability", () => {
     });
   });
 
-  it("determines if nodeType is applicability", () => {
```

Comment:
How would we achieve testing of a private function?

Reply:
you'd call the hook that calls the private function, then verify the side effects of the private function

Reply:
as discussed in slack, these tests might not be relevant anymore since the result being returned -> isApplicability returned true, as long as we have a test for negative case as well, this is fine